app/src/main/java/com/zeapo/pwdstore/UserPreference.kt

```diff
@@ -358,13 +355,24 @@ class UserPreference : AppCompatActivity() {
             }
         }
 
+        private fun updatePgpKeyPreference(): String = with(findPreference<Preference>("")) {
```

Comment:
Is the empty string correct here?

Reply:
No, I pushed a commit later, you're looking at an outdated diff.